cmake/projects/Qt/schemes/url_sha1_qt.cmake.in

```diff
@@ -215,6 +215,9 @@ if(BUILD_SHARED_LIBS)
   list(APPEND configure_opts "-shared")
 else()
   list(APPEND configure_opts "-static")
+  if(NOT QT_BUILD_DYNAMIC_VSRUNTIME)
+    list(APPEND configure_opts "-static-runtime")
```

Comment:
Linux build failed with message: `-static-runtime: invalid command-line switch`. I guess what I was afraid [will happen with Boost](https://github.com/ruslo/hunter/pull/1224#discussion_r153008429) happened with Qt :) We need `if(MSVC)` protection here.